<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>82</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QPushButton" name="pButton_vote">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Version &amp; Vote</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="scoreAgeLabel">
     <property name="text">
      <string>0/0</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="quarantineLabel">
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="kisstester.qrc">:/icons/wait.png</pixmap>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="hasvotesLabel">
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="kisstester.qrc">:/icons/pass.png</pixmap>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="unlockedLabel">
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="kisstester.qrc">:/icons/padlock_open.png</pixmap>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pButton_bug">
     <property name="text">
      <string>Report bug</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pButton_pdetails">
     <property name="text">
      <string>Details</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="kisstester.qrc"/>
 </resources>
 <connections/>
</ui>
